### PR TITLE
Added filter for hidden commands and options

### PR DESCRIPTION
**Pull request by GhostOps77**
Changes done in this branch:

- Merged code with PR #65
- Added ignore C901 for flake8 (or else, the PR will fail, for the reason that `ClickCompleter.get_completions` is too complex)
- Added test cases for hidden command & options in tests/test_basic.py

### CODE REVIEW

**Comment by GhostOps77** [comment]
@auvipy  waiting for ur review man

File: click_repl/__init__.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ class ClickExit(RuntimeError):
     text_type = str  # noqa
 
 
-__version__ = "0.2.0"
+__version__ = "0.2.1"
```
Comment (auvipy) [inline click_repl/__init__.py:39]: do we need this now?

Comment (GhostOps77) [reply click_repl/__init__.py]: i swear, there must be a dev branch for these type of changes
so that i can give 0.2.1dev as the version

**Review by auvipy — APPROVED** [review]

**Comment by GhostOps77** [comment]
Solves issue: #64

**Pull request merged**